server.go

```diff
@@ -121,15 +217,25 @@ func (server *Server) panicMiddleware(next http.Handler) http.Handler {
 	})
 }
 
-// pingRoute response with status of load balancer server itself
+// pingRoute returns status of waggle server itself
 func (server *Server) pingRoute(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	response := PingResponse{Status: "ok"}
 	json.NewEncoder(w).Encode(response)
 }
 
-// signDropperRoute response with status of load balancer server itself
+// signDropperRoute sign dropper call requests
 func (server *Server) signDropperRoute(w http.ResponseWriter, r *http.Request) {
+	authorizationToken := r.Context().Value("authorizationToken").(string)
+
+	isMetatxDrop := false
+	queries := r.URL.Query()
+	for k := range queries {
```

Comment:
Instead of this `range queries` loop, you can use `queries.Get("is_metatx_drop")`. It will be empty string if the query parameter is not specified and nonempty otherwise.

Reply:
https://pkg.go.dev/net/url#Values.Get

Reply:
Actually [`queries.Has`](https://pkg.go.dev/net/url#Values.Has) would be even better.